aws-py-ecs-instances-autoapi/py-ecs-instance/Pulumi.yaml

```
@@ -0,0 +1,3 @@
+name: aws-py-ecs-instances-autoapi
+description: A container running in AWS ECS Container Instance, using Python infrastructure as code
+runtime: python
```

I tried this out locally and running `yarn start` fails for me because none of the dependencies for the Python program are installed.

This can be addressed with the following change:

```suggestion
runtime:
  name: python
  options:
    virtualenv: venv
```

With that set in `Pulumi.yaml` and CLI version 2.15.1 or higher, if the directory specified in the `virtualenv` option is empty or doesn't exist, the virtual environment will be created and deps installed into it as part of running `refresh/preview/up`.